src/plugins/data/public/plugin.ts

```diff
@@ -52,6 +60,7 @@ export class DataPublicPlugin implements Plugin<DataPublicPluginSetup, DataPubli
       autocomplete: this.autocomplete,
       getSuggestions: getSuggestionsProvider(core.uiSettings, core.http),
       search: this.searchService.start(core),
+      fieldFormats: this.fieldFormats,
```

Comment:
registry manipulation is possible in the start method. we should only allow registration in the setup phase.

Reply:
Fixed